CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 #### 🚀 Enhancement
 
-- update .gitignore [#21](https://github.com/JupiterOne/graph-malwarebytes/pull/21) ([@dobregons](https://github.com/dobregons))
-- Upgrade to modern patterns [#20](https://github.com/JupiterOne/graph-malwarebytes/pull/20) ([@VDubber](https://github.com/VDubber) [@dobregons](https://github.com/dobregons))
+- update .gitignore
```

Comment:
Can this file get a touch of cleanup? Line 20 typically is at the top of the file.